client/src/pages/Popular.tsx

```diff
@@ -8,6 +8,8 @@ import Header from '../components/Header';
 import axios from 'axios';
 
 import '../styles/Popular.css';
+import '../styles/Navbar.css';
+import '../styles/ProductGrid.css'
```

Comment:
Will the page render correctly without these imports? They should already be imported in the corresponding `Navbar` and `ProductGrid` components.